src/npc.cpp

```diff
@@ -629,7 +630,7 @@ int NpcScriptInterface::luaActionMoveTo(lua_State* L)
 	// selfMoveTo(x, y, z[, minTargetDist = 1[, maxTargetDist = 1[, fullPathSearch = true[, clearSight = true[,
 	// maxSearchDist = 0]]]]]) selfMoveTo(position[, minTargetDist = 1[, maxTargetDist = 1[, fullPathSearch = true[,
 	// clearSight = true[, maxSearchDist = 0]]]]])
-	Npc* npc = tfs::lua::getScriptEnv()->getNpc();
+	auto npc = tfs::lua::getScriptEnv()->getNpc();
```

Comment:
Could this be `const auto`
```suggestion
	const auto npc = tfs::lua::getScriptEnv()->getNpc();
```